Source/Csla/DataPortalClient/HttpProxy.cs

```diff
@@ -173,6 +187,12 @@ private async Task<byte[]> CallViaHttpClient(byte[] serialized, string operation
 
     private byte[] CallViaWebClient(byte[] serialized, string operation, string routingToken)
```

Comment:
Shouldn't we mark the method also with unspportedOS?

Reply:
That would bubble problem to callsite, which is select between sync/async methods. Currently compiler analyze that if we on platform "browser" then we throw PNSE and that's valid way to "support" something.

Given that if I mark this method as unsupported, I have to perform check when `sync == true` and I still have to throw PNSE there. OR! If you think that always use async versions in the browser is valid idea, that maybe reasonable.

Reply:
The question is: Do `sync == true` csla operations currently work with blazor? As far as I understand these changes here it should _not_ work. So we can _assume_ async is the _only_ way in blazor/browser.
@rockfordlhotka You have experience with CSLA + Blazor. Do you use sync methods anywhere?

Reply:
Via `HttpProxy`? Yes, it would only be async in Blazor.

I could see where _LocalProxy_ might be used from a sync method, but never `HttpProxy`.

Reply:
I preserve current behaviour, since WebClient does not supported on the browser and blow up in runtime.

Reply:
Okay. Maybe we should create an analyzer to detect sync calls in a browser context? Is that even possible? 🤔

Reply:
I’m not sure that it would be interesting. Maybe if we have additional methods for sync calls. Otherwise we are in the data-flow business and I’m not confident that this is worth it. 

Isn’t this just blowup during regular dev flow? I think I can add proper messages to indicate that this is limitations Blazor/browser and maybe point to some wiki page to make this actionable to developers

Reply:
You are probably right. We should have a simple message in the PNSE like `Sync data portal methods are not supported in a browser enviroment. Please use the async variant instead.`